dags/glam_fenix.py

```diff
@@ -18,6 +18,7 @@
     "email_on_retry": True,
     "retries": 1,
     "retry_delay": timedelta(minutes=30),
+    "max_active_runs": 1
```

Comment:
I believe this needs to be passed as [parameter into `DAG(... , max_active_runs=1, ...)`](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/models/dag/index.html#airflow.models.dag.DAG). See also: https://github.com/mozilla/telemetry-airflow/blob/89724725eef46d2a7f9c75f93d8af1da9eb7e990/dags/experiments_live.py#L22